services/proxy-api/serverless.yml

```diff
@@ -0,0 +1,189 @@
+  atomicCounterTableArn: ${env:AMENDMENTS_COUNTER_TABLE_ARN, cf:database-${self:custom.stage}.AmendmentsAtomicCounterTableArn}
+  webAclName: ${self:service}-${self:custom.stage}-webacl
+  associateWaf:
+    name: ${self:custom.webAclName}
+    version: V2
+  warmup:
+    default:
+      enabled: true
+      prewarm: true
+      events:
+        - schedule: rate(${ssm:/configuration/${self:custom.stage}/warmup/schedule~true, ssm:/configuration/default/warmup/schedule~true, "4 minutes"})
+      concurrency: ${ssm:/configuration/${self:custom.stage}/warmup/concurrency~true, ssm:/configuration/default/warmup/concurrency~true, 5}
+  scripts:
+    hooks:
+      package:initialize: |
+        set -e
+        curl -L --output lambda_layer.zip https://www.princexml.com/download/prince-14.2-aws-lambda.zip && zip lambda_layer.zip etc/fontconfig/fonts.conf
+provider:
+  name: aws
+  runtime: nodejs12.x
+  region: us-east-1
+  tracing:
+    apiGateway: true
+  logs:
+    restApi: true
+  iam:
+    role:
+      path: ${ssm:/configuration/${self:custom.stage}/iam/path~true, ssm:/configuration/default/iam/path~true, "/"}
+      permissionsBoundary: ${ssm:/configuration/${self:custom.stage}/iam/permissionsBoundaryPolicy~true, ssm:/configuration/default/iam/permissionsBoundaryPolicy~true, ""}
+      statements:
+        - Effect: "Allow"
+          Action:
+            - dynamodb:DescribeTable
+            - dynamodb:Query
+            - dynamodb:Scan
+            - dynamodb:GetItem
+            - dynamodb:PutItem
+            - dynamodb:UpdateItem
+            - dynamodb:DeleteItem
+          Resource:
+            - ${self:custom.tableArn}
+            - ${self:custom.atomicCounterTableArn}
+  environment:
+    tableName: ${self:custom.tableName}
+    atomicCounterTableName: ${self:custom.atomicCounterTableName}
+
+functions:
+  create:
+    handler: handlers/create.main
+
+    events:
+      - http:
+          path: amendments
+          method: post
+          cors: true
+          authorizer: aws_iam
+  get:
+    handler: handlers/get.main
+    events:
+      - http:
+          path: amendments/{id}
+          method: get
+          cors: true
+          authorizer: aws_iam
+  list:
+    handler: handlers/list.main
+    events:
+      - http:
+          path: amendments
+          method: get
+          cors: true
+          authorizer: aws_iam
+  update:
+    handler: handlers/update.main
+    events:
+      - http:
+          path: amendments/{id}
+          method: put
+          cors: true
+          authorizer: aws_iam
+  delete:
+    handler: handlers/delete.main
+    events:
+      - http:
+          path: amendments/{id}
+          method: delete
+          cors: true
+          authorizer: aws_iam
+  princeProxy:
+    handler: handlers/princeProxy.main
+    environment:
+      princeApi: ${self:custom.princeApi}
+    timeout: 30
+    memorySize: 3008
+    maximumRetryAttempts: 0
+    events:
+      - http:
+          path: princeProxy
```

Comment:
Since this service just houses the proxy, please strip out all of the stuff that belongs in `services/app-api` IE `list`, `update`, `delete` etc. Also please change `princeProxy` to `proxy` or similar, we want this to agnostic and not specific to Prince. 

Stripping out this stuff will get rid of most if not all of those code climate findings as well.